jar-infer/jar-infer-lib/src/main/java/com/uber/nullaway/jarinfer/DefinitelyDerefedParamsDriver.java

```diff
@@ -225,6 +236,8 @@ private void analyzeFile(String pkgName, String inPath)
           // Only process classes in specified classpath and not its dependencies.
           // TODO: figure the right way to do this
           if (!pkgName.isEmpty() && !cls.getName().toString().startsWith(pkgName)) continue;
+          // Skip non-public / ABI classes
+          if (!cls.isPublic() && !includeNonPublicClasses) continue;
```

Comment:
This line is basically the only real change, the rest is needed in order for test to keep working :)

Reply:
Sounds good.